package.json

```diff
@@ -61,6 +61,7 @@
     "pack-client": "npm-run-all -p -l pack-client-*",
     "test-client-template": "cd sdk/template/template && npm run test",
     "test-client": "npm-run-all -p -l \"test-client-* -- {@}\"",
+    "test-client-eventhubs-client": "echo skipped",
```

Comment:
For consistency, please add similar scripts for all packages (`eventhubs-processor`, `keyvault`, etc).